actionmailer/lib/action_mailer/base.rb

```diff
@@ -163,7 +163,7 @@ module ActionMailer
   #
   # Multipart messages can also be used implicitly because Action Mailer will automatically detect and use
   # multipart templates, where each template is named after the name of the action, followed by the content
-  # type. Each such detected template will be added as a separate part to the message.
+  # type. Each detected template will be added to the message, as a separate part.
```

Comment:
Just retain the "such", rest is good. @tomkadwill if we read a line before and this together, they both are linked and should maintain continuity.